omnigibson/robots/manipulation_robot.py

```diff
@@ -294,7 +294,8 @@ def _find_gripper_contacts(self, arm="default", return_contact_positions=False):
             return {other for other, _ in raw_contact_data}, robot_contact_links
 
         # Otherwise, we rely on the simpler, but more costly, get_contact_data API.
-        contacts = GripperRigidContactAPI.get_contact_data(self.scene.idx, column_prim_paths=link_paths)
+        finger_paths = set([link.prim_path for link in self.finger_links[arm]])
+        contacts = GripperRigidContactAPI.get_contact_data(self.scene.idx, column_prim_paths=finger_paths)
```

Comment:
you should try to merge in the changes from my PR, which should include everything you have here.